File: decimal to binary.cpp
#include<iostream>
using namespace std;

int main(){
    int num, binary = 0, remainder = 0, product = 1;
    cout<<"Enter a number to convert into binary: ";
    cin>>num;
    while(num != 0){
        remainder = num % 2;
        binary = binary + (remainder * product);
        num /= 2;
        product *= 10;
    }
    cout<<"The binary form is: "<<binary;
}

File: decimal to binary.py
remainder = 0
position = 1
binary = 0

decimal = int(input("Enter a number: "))
temp = decimal

while temp != 0:
    remainder = temp % 2
    temp //= 2
    binary = binary + (position * remainder)
    position *= 10

print(f"The binary form of {decimal} = {binary}")